docs/api/routing.rst

```diff
@@ -394,6 +394,61 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other 
+paradigms. Consider the task of building an API for a calculator which can both add and subtract 
+two numbers. You could implement the 
+following:
+
+.. code:: python
+
+    class Add():
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) + int(req.get_param('y')))
+            resp.status = falcon.HTTP_200
+
+    class Subtract():
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) - int(req.get_param('y')))
```

Comment:
We should use [`Response.text`](https://falcon.readthedocs.io/en/latest/api/request_and_response_wsgi.html#falcon.Response.text) in lieu of the now-deprecated [Response.body](https://falcon.readthedocs.io/en/latest/api/request_and_response_wsgi.html#falcon.Response.body).

Reply:
Updated